indexing-service/src/main/resources/indexer_static/js/console-0.0.1.js

```diff
@@ -2,39 +2,56 @@
 
 var oTable = [];
 
+var killTask = function(taskId) {
+  if(confirm('Do you really want to kill: '+taskId)) {
+    $.ajax({
+      type:'POST',
+      url: '/druid/indexer/v1/task/'+ taskId +'/shutdown',
+      data: ''
+    }).done(function(data) {
+      alert('Submitted kill request for: '+data.task);
```

Comment:
@drcrallen added refresh logic...timeout is present to let metadata store get updated with the new task status otherwise if refresh is done immediately the task will still show up in the original list.